docs/reference/how-to/search-speed.asciidoc

```diff
@@ -9,6 +9,27 @@ fast. In general, you should make sure that at least half the available memory
 goes to the filesystem cache so that Elasticsearch can keep hot regions of the
 index in physical memory.
 
+[discrete]
+=== Avoid page cache thrashing by using modest read ahead values on Linux
+
+Search can cause a lot of randomized read IO. When using the default
+Elasticsearch filestore (fs/<<hybridfs,hybridfs>>), Lucene files using memory
+mapping may contribute to unnecessary read I/O when the underlying block device
+has a high readahead value.
+
+Most Linux distributions use a sensible readahead value of `128KiB` for a
+single plain device, however, when using software raid, LVM or dm-crypt the
+resulting block device (backing Elasticsearch <<path-settings,path.data>>)
+may end up having a very large readahead value (in the range of several MiB).
+This usually results in severe page (filesystem) cache thrashing adversely
+affecting search (or <<docs,update>>) performance.
+
+You can check the current value using `lsblk -o NAME,RA,MOUNTPOINT,TYPE,SIZE`.
```

Comment:
Maybe add a hint that `RA` denotes the readahead value in KiB?

Reply:
Addressed in e7e8609d2d72475f979022fe702930ae6937f52f